### DISCUSSION TITLE
PICO8 Images Folder / Amiga Name on Consoles

### DISCUSSION THREAD

**Discussion by vics3g** [OnionUI/Onion#522] [General]
I have put on PICO Folder ROMS and Images in Imgs, and these folder still appearing on Console Menu. 

And Amiga name appear like:  aemgs or somthing like that.

How i can change systems name on menu?

**Comment by sectune** [comment]
Pico8 games are images all by themselves, Imgs folder support would still be nice though in case user wants to switch image to a custom desired one.
